internal/data/fixtures.go

```diff
@@ -0,0 +1,3 @@
+package data
+
+const InsertPaymentsQuery = `INSERT INTO ingest_payments (operation_id, operation_type, transaction_id, transaction_hash, from_address, to_address, src_asset_code, src_asset_issuer, src_amount, dest_asset_code, dest_asset_issuer, dest_amount, created_at, memo) VALUES (:operation_id, :operation_type, :transaction_id, :transaction_hash, :from_address, :to_address, :src_asset_code, :src_asset_issuer, :src_amount, :dest_asset_code, :dest_asset_issuer, :dest_amount, :created_at, :memo);`
```

Comment:
I'd recommend we work with functions instead of providing queries for fixtures because using functions is easier. Additionally, it's easy to maintain. For example, I as a developer would need to look a this query every time to understand what field I need to pass to it. Also, it's not clear that this is used only for testing purposes, someone would misunderstand and use it on the actual code.

This is not a blocker, it's just that we need to define a pattern.

Reply:
[fixture reorganized](https://github.com/stellar/wallet-backend/pull/28/commits/77e0be4d68fccbfa99174ab67fd51955b08524fb)